tests/utils.go

```diff
@@ -706,3 +707,38 @@ func createNodeDaemonSet(ds *appsv1.DaemonSet) {
 		gomega.BeNil(),
 		"creating node plugin daemonset %v", nodeDaemonSet)
 }
+
+func enableThinpoolMonitoring() {
+	lv := VOLGROUP + "/" + pvcObj.Spec.VolumeName
+	vol, err := lvm.ListLVMLogicalVolume(lv)
+
+	gomega.Expect(err).To(gomega.BeNil(), "list LVM LV")
+	gomega.Expect(len(vol)).To(gomega.Equal(1), "retrieve LVM LV")
+
+	thinpool := VOLGROUP + "/" + vol[0].PoolName
+
+	args := []string{
+		"--monitor", "y",
+	}
+
+	err = lvm.ChangeLVMLogicalVolumeAttrs(thinpool, args)
```

Comment:
shouldn't this be enabled by the product itself?

Reply:
Enabling by default in product is a tricky decision depending on who wants it and who doesn't. Some discussion here: https://github.com/openebs/lvm-localpv/issues/93

Reply:
I didn't mean by default right. I meant that product should add by itself, depending on user configuration.
But from this I take it that today the product would never do that, and users have to manually enable it?
Then it's fine to test it as such.

Reply:
> But from this I take it that today the product would never do that

That's right, currently we have no provision to do that. There are a couple of enhancements identified in this area.